addon/components/cells/base.js

```diff
@@ -21,15 +20,10 @@ const Cell = Component.extend({
   layout,
   tagName: 'td',
   classNames: ['lt-cell'],
-  attributeBindings: ['style'],
   classNameBindings: ['align', 'isSorted', 'column.cellClassNames'],
 
   isSorted: computed.readOnly('column.sorted'),
 
-  style: computed('column.width', function() {
```

Comment:
Are we concerned that this no longer happens when `enableScaffolding` is set to false?